### PR TITLE
Fix doc typos 5

### PR DESCRIPTION
**Pull request by drwells**
The quest continues.

### CODE REVIEW

**Comment by tamiko** [comment]
At least I was consistently wrong [1]

[1] http://cdn.shopify.com/s/files/1/0535/6917/products/consistencydemotivator_1024x1024.jpeg?v=1414004030

File: include/deal.II/base/quadrature_lib.h [hunk 1]
```diff
@@ -157,7 +157,7 @@ class QSimpson : public Quadrature<dim>
  *
  * The class is poorly named since the proper name of the quadrature formula
  * is "trapezoidal rule", or sometimes also called the "trapezoid rule". The
- * misnomer results from the fact that its original authors's poor English
+ * misnomer results from the fact that its original authors' poor English
```
Comment (bangerth) [inline include/deal.II/base/quadrature_lib.h]: Well, this one is actually too funny :-)

Comment (drwells) [inline include/deal.II/base/quadrature_lib.h]: It was *extremely tempting* to leave that one in. I was entertained.

Comment (bangerth) [inline include/deal.II/base/quadrature_lib.h]: :-)

**Pull request merged**